### PR TITLE
Use new footnote syntax in `History of osu!/2014`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/History_of_osu!/2014/en.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ Additionally, [peppy](https://osu.ppy.sh/users/2) also released the "osu!test" b
...
+[^all-gamemodes]: [osu! newspost by Tom94 (1 March 2014) - "Performance ranking for all gamemodes"](https://osu.ppy.sh/home/news/2014-03-01-performance-ranking-for-all-gamemodes)
+[^change-background]: [osu! forum comment (Resolved Issues) by Pawsu - "[resolved] How to change all beatmap backgrounds")](https://osu.ppy.sh/community/forums/topics/397827?start=4749640)
+[^skinning]: [osu! wiki article - "Skinning history"](/wiki/Skinning/History)
+[^fanart]: [osu! newspost by LuigiHann (3 April 2014) - "osu! Monthly Fanart Contest #1 - Ends this week!"](https://osu.ppy.sh/home/news/2014-04-03-osu-monthly-fanart-contest-1-ends-this-week)
+[^fanart-results]: [osu! newspost by LuigiHann (21 May 2014) - "First Official osu! Fanart Contest Results!"](https://osu.ppy.sh/home/news/2014-05-21-first-official-osu-fanart-contest-results)
+[^download]: [osu! newspost by peppy (18 June 2014) - "Download restrictions removed"](https://osu.ppy.sh/home/news/2014-06-18-download-restrictions-removed)
+[^bat]: [osu! newspost by Ephemeral (21 August 2014) - "Restructuring of the BAT"](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat)
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2014/en.md:44]: should conform the formatting according to the examples in the asc https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#citations

File: wiki/History_of_osu!/2014/en.md [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
 
 ## March
 
-On 1 March 2014 the performance points system redesign by Tom94 in months prior was ready to be set live for [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania) as the official ranking system for those modes, along with a reminder that the current system was not yet 100% finished, and further feedback from players was still needed \[4\].
+On 1 March 2014 the performance points system redesign by Tom94 in months prior was ready to be set live for [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania) as the official ranking system for those modes, along with a reminder that the current system was not yet 100% finished, and further feedback from players was still needed.[^all-gamemodes]
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2014/en.md:9]:
```suggestion
On 1 March 2014 the performance points system redesign by Tom94 in months prior was ready to be set live for [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), and [osu!mania](/wiki/Game_mode/osu!mania) as the official ranking system for those modes, along with a reminder that the current system was not yet 100% finished, and further feedback from players was still needed.[^all-game-modes]
```

File: wiki/History_of_osu!/2014/en.md [hunk 3]
```diff
@@ -2,27 +2,27 @@
...
 
-As [skinning](/wiki/Skinning) elements expanded, the element, `playfield.jpg` was deprecated and replaced with a new generic one that is not modifiable. `playfield.jpg` was an element that allowed skinners to use any image as the default background of a beatmap if one was not supplied. This functionality was later brought in the form of `menu-background.jpg`, which changed both the main screen and default playfield backgrounds \[5,6\]. <!--date needed-->
+As [skinning](/wiki/Skinning) elements expanded, the element, `playfield.jpg` was deprecated and replaced with a new generic one that is not modifiable. `playfield.jpg` was an element that allowed skinners to use any image as the default background of a beatmap if one was not supplied. This functionality was later brought in the form of `menu-background.jpg`, which changed both the main screen and default playfield backgrounds.[^change-background][^skinning] <!--date needed-->
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2014/en.md:11]:
```suggestion
As [skinning](/wiki/Skinning) elements expanded, the element, `playfield.jpg` was deprecated and replaced with a new generic one that is not modifiable. `playfield.jpg` was an element that allowed skinners to use any image as the default background of a beatmap if one was not supplied. This functionality was later brought in the form of `menu-background.jpg`, which changed both the main screen and default playfield backgrounds.[^change-background][^skinning]<!--date needed-->
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/2014/en.md [hunk 4]
```diff
@@ -2,43 +2,43 @@
...
-5. [osu! forum comment (Resolved Issues) [Pawsu] - "[resolved] How to change all beatmap backgrounds")](https://osu.ppy.sh/community/forums/topics/397827?start=4749640)
-6. [osu! wiki article - "Skinning history"](/wiki/Skinning/History)
-7. [osu! newspost [LuigiHann] (3 April 2014) - "osu! Monthly Fanart Contest #1 - Ends this week!"](https://osu.ppy.sh/home/news/2014-04-03-osu-monthly-fanart-contest-1-ends-this-week)
-8. [osu! newspost [LuigiHann] (21 May 2014) - "First Official osu! Fanart Contest Results!"](https://osu.ppy.sh/home/news/2014-05-21-first-official-osu-fanart-contest-results)
-9. [osu! newspost [peppy] (18 June 2014) - "Download restrictions removed"](https://osu.ppy.sh/home/news/2014-06-18-download-restrictions-removed)
-10. [osu! newspost [Ephemeral] (21 August 2014) - "Restructuring of the BAT"](https://osu.ppy.sh/home/news/2014-08-21-restructuring-of-the-bat)
+[^ranking-maintenance]: [osu! newspost by peppy (2014-01-20) "Performance Ranking Maintenance"](https://osu.ppy.sh/home/news/2014-01-20-performance-ranking-maintenance)
```
Comment (The-Last-Cookie) [inline wiki/History_of_osu!/2014/en.md:35]: I've outdated the translations to make the contributors aware of these lines specifically.

**Review by TicClick — APPROVED** [review]

**Pull request merged**